index.js

```diff
@@ -1,70 +1,34 @@
-const denodeify = require('then-denodeify')
-const nodeify = require('then-nodeify')
+module.exports = function createMediator() {
+	const providers = Object.create(null)
```

Comment:
So that it can be safely used as a map - normal objects have a prototype Object, which means that `mediator.call('constructor')` would totally do something even if you had't added a provider for it yet.